sdk/core/azure-core/src/main/java/com/azure/core/implementation/RestProxy.java

```diff
@@ -505,7 +507,7 @@ protected Object handleHttpResponse(final HttpRequest httpRequest, Mono<HttpDeco
 
     protected Object handleResumeOperation(HttpRequest httpRequest, OperationDescription operationDescription, SwaggerMethodParser methodParser, Type returnType, Context context)
         throws Exception {
```

Comment:
If we keep the change to throwing `RuntimeException` you'll need to remove this checked throw since RuntimeExceptions can be unchecked.